<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HostMemoryRow</class>
 <widget class="QWidget" name="HostMemoryRow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Host Memory</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="hostNameLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Host Name</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="contentLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="HostShinyBar" name="hostShinyBar" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="memoryDetailsGroup">
       <property name="title">
        <string>Memory Details</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="horizontalSpacing">
         <number>8</number>
        </property>
        <property name="verticalSpacing">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelTotal">
          <property name="text">
           <string>Total:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="valueTotal">
          <property name="text">
           <string>0 GB</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelUsed">
          <property name="text">
           <string>Used:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="valueUsed">
          <property name="text">
           <string>0 GB</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelAvailable">
          <property name="text">
           <string>Available:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="valueAvailable">
          <property name="text">
           <string>0 GB</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelTotDynMax">
          <property name="text">
           <string>Total Dynamic Maximum:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="valueTotDynMax">
          <property name="text">
           <string>0 GB</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelOvercommit">
          <property name="text">
           <string>Overcommit:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="labelOvercommitValue">
          <property name="text">
           <string>0%</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="labelControlDomain">
          <property name="text">
           <string>Control Domain:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="valueControlDomain">
          <property name="text">
           <string>0 GB</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HostShinyBar</class>
   <extends>QWidget</extends>
   <header>widgets/hostshinybar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
